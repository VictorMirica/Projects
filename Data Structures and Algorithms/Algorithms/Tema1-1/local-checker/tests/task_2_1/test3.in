0001011101100101