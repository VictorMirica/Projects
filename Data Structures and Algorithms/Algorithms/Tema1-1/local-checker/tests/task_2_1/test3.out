XXXXXXXXXXXXXXXX0001011101100101